### DISCUSSION TITLE
Example of cFE funcitonal test?

### DISCUSSION THREAD

**Discussion by ahatstat** [nasa/cFS#717] [Q&A]
[This doc](https://github.com/nasa/cFE/blob/b72dd4e1f9f44c7dbb7a12895b5ac1635eb239b2/docs/README_functionaltest.md) hints that its possible to use the UT_Assert framework to write functional tests for a set of mission apps.  Does one need to create a special test app? It's not clear to me from the doc how to implement this.  Is there an example of how to do this somewhere?

**Comment by skliper** [comment]
cFE has a test app that utilizes UT_Assert:
https://github.com/nasa/cFE/tree/b72dd4e1f9f44c7dbb7a12895b5ac1635eb239b2/modules/cfe_testcase
